src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core/Environment.cs

```diff
@@ -45,15 +44,22 @@ public Environment(ISystemInfo systemInfo, IProcessStatic processStatic, IConfig
                 AddVariable("Product Name", () => fileVersionInfo?.ProductName);
 
                 AddVariable("OS", () => System.Environment.OSVersion?.VersionString);
+
+                // report operating system and version in addition to "OS" - these two report more useful information (including linux distro names)
+                // Eventually, we should deprecate "OS" in favor of these variables
+                AddVariable("Operating System", () => RuntimeEnvironmentInfo.OperatingSystem);
```

Comment:
Can you give some examples of what this will look like?

Reply:
A couple of examples: 
![image](https://user-images.githubusercontent.com/120425148/224801859-b99522f8-5283-457a-b1c3-05aa0fffbf67.png)
![image](https://user-images.githubusercontent.com/120425148/224801928-4158ddca-87c0-472d-900a-acba65583de8.png)